test/controllers/trades_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class TradesControllerTest < ActionController::TestCase
+
+  def setup
+    @controller = TradesController.new
+  end
+
+  test "create method should update database entries" do
+    create_start_of_game
+    ApplicationController.any_instance.stubs(:in_game).returns(true)
+
+    item_skill_set = SkillSet.create(strength: 1,
+                                    dexterity: 2,
+                                 intelligence: 3,
+                                       health: 4,
+                                        money: -3,
+                                        speed: 6)
+
+    item1 = Item.create(name: "dagger", skill_set: item_skill_set)
+    item2 = Item.create(name: "bowling ball", skill_set: item_skill_set)
+    item3 = Item.create(name: "basket", skill_set: item_skill_set)
+    @character.items << [item1, item2]
+    @store.items << item3
+
+    params = {
+              "classes"=>
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Surrounding space missing for operator `=>`.